apps/styleguide/src/forms/InputInt.vue

```diff
@@ -1,47 +1,99 @@
+<template>
```

Comment:
And also for this file?

Reply:
InputDecimal and InputInt used to inherit InputString for its behavior. The expression toolkit needs the numbers to be of a number type and not a string type. I tried to work around this with _baseInput's parser but that gives lots of issues. So now the number and int fields are there own components that don't inherit from InputString. This seems to work fine.